### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by Mucyosoda**
Desktop version  and responsive 

-  Added index.html
- Added style.css
- Added linters
- Added gitignore
- Added readme file

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @Mucyosoda,

**This is a great project you have going on, well-done lad**. 

   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️
_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -8,47 +8,355 @@
...
+          <div class="my_recent">
+            <h1>My Recent Works</h1>
+          </div>
+          <div class="line1"></div>
+        </div>
+        <div class="txt_image">
+          <div class="img1">
+            <img src="images/Img.png" alt="image" class="middle-image" />
+          </div>
+          <div class="post1">
```
Comment (mke2111) [inline index.html:77]: - [ ] For this section, align both the left and right sides to the top.
Screenshot from 2021-08-20 12-28-55

File: index.html [hunk 2]
```diff
@@ -8,47 +8,355 @@
...
+                  <li><a href="#"></a>html</li>
+                  <li class="boot2"><a href="#"></a>bootstrap</li>
+                  <li><a href="#"></a>Ruby</li>
+                </ul>
+              </div>
+            </div>
+            <div class="cardyellow"><h4>See project</h4></div>
+          </div>
+        </div>
+        <div id="content_one">
```
Comment (mke2111) [inline index.html:208]: - [ ] The about section is being overflowed by the projects section. Give it a fix.

File: index.html [hunk 3]
```diff
@@ -8,47 +8,355 @@
...
+              <li><a href="#"></a>css</li>
+              <li><a href="#"></a>html</li>
+              <li class="boot"><a href="#"></a>bootstrap</li>
+              <li><a href="#"></a>Ruby</li>
+            </ul>
+            <button type="submit">See Project</button>
+          </div>
+        </div>
+
+        <div class="cards_room">
```
Comment (mke2111) [inline index.html:95]: Screenshot from 2021-08-20 12-37-06

File: index.html [hunk 4]
```diff
@@ -8,47 +8,355 @@
...
+                  <li class="boot2"><a href="#"></a>bootstrap</li>
+                  <li><a href="#"></a>Ruby</li>
+                </ul>
+              </div>
+            </div>
+            <div class="cardyellow"><h4>See project</h4></div>
+          </div>
+        </div>
+        <div id="content_one">
+          <div class="content_3">
```
Comment (mke2111) [inline index.html:209]: Screenshot from 2021-08-20 12-37-06

File: index.html [hunk 5]
```diff
@@ -8,47 +8,355 @@
...
+
+      <div class="footer_form">
+        <div class="content_4">
+          <p>
+            I'm always interested in <br />
+            hearing about new projects, <br />so if you'd like to<br />
+            chat please get in.
+          </p>
+        </div>
+        <div class="form_view">
```
Comment (mke2111) [inline index.html:278]: - [ ] This section should cover the entire space and equal the section above.
Screenshot from 2021-08-20 12-43-56

File: style.css [hunk 6]
```diff
@@ -1,99 +1,727 @@
...
+
+.social-icons_last a {
+  color: #505f79;
+}
+
+.form_view1 {
+  display: none;
+}
+
+@media only screen and (min-width: 768px) {
```
Comment (mke2111) [inline style.css:383]: - [ ] The value for min-width property should be in range of 992px - 1100px. These are the accepted desktop screen breakpoints. Change your value. What you are using lies under the tablet screen breakpoints.

File: style.css [hunk 7]
```diff
@@ -1,99 +1,727 @@
...
+.bottom_link li {
+  margin-top: 15px;
+  font-size: 20px;
+}
+
+.content_3 {
+  padding: 40px;
+}
+
+form {
```
Comment (mke2111) [inline style.css:303]: - [ ] For mobile screen sizes, I believe a bug has been introduced while working on the desktop version. The contact form display has been altered for mobile screens.
Screenshot from 2021-08-20 12-48-29

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @Mucyosoda, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 8]
```diff
@@ -8,47 +8,355 @@
...
+          <h1>My Recent Works</h1>
+        </div>
+        <div class="line1"></div>
+      </div>
+      <div id="sidebar">
+        <div class="txt_image">
+          <div class="img1">
+            <img src="images/Img.png" alt="image" class="middle-image" />
+          </div>
+          <div class="post1">
```
Comment (mke2111) [inline index.html:77]: > For this section, align both the left and right sides to the top.
Screenshot from 2021-08-20 12-28-55

Please consider aligning this section as previously requested.

File: index.html [hunk 9]
```diff
@@ -8,47 +8,355 @@
...
+                  <li class="boot2"><a href="#"></a>bootstrap</li>
+                  <li><a href="#"></a>Ruby</li>
+                </ul>
+              </div>
+            </div>
+            <div class="cardyellow"><h4>See project</h4></div>
+          </div>
+        </div>
+        <div id="content_one">
+          <div class="content_3">
```
Comment (mke2111) [inline index.html:209]: > The about section is being overflowed by the projects section. Give it a fix.
Screenshot from 2021-08-20 12-37-06

Please consider fixing the overflow issue

File: index.html [hunk 10]
```diff
@@ -8,47 +8,355 @@
...
+
+      <div class="footer_form">
+        <div class="content_4">
+          <p>
+            I'm always interested in <br />
+            hearing about new projects, <br />so if you'd like to<br />
+            chat please get in.
+          </p>
+        </div>
+        <div class="form_view">
```
Comment (mke2111) [inline index.html:278]: > This section should cover the entire space and equal the section above.
Screenshot from 2021-08-20 12-43-56

Please consider stretching out this section to fit and match the other sections. Focus mainly on its padding/margin

File: index.html [hunk 11]
```diff
@@ -8,47 +8,324 @@
       rel="stylesheet"
       href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
     />
+    <title>Desktop version</title>
     <link rel="stylesheet" href="style.css" />
-    <title>Setup and mobile version</title>
   </head>
   <body>
```
Comment (SajjadAhmad14) [inline index.html:14]: [OPTIONAL] Please remove space from the right and left of the page and follow the design exactly. Kindly see the below screenshot for reference. 
space

File: index.html [hunk 12]
```diff
@@ -8,47 +8,324 @@
...
-    <header>
-      <nav class="nav">
-        <h2 class="name">Mucyo</h2>
-
-        <i class="fa fa-bars"></i>
+    <div class="container">
+      <nav>
+        <div class="logo">LOGO</div>
```
Comment (SajjadAhmad14) [inline index.html:17]: [ OPTIONAL ] Please align navigation items vertically, and use the font which is defined in the design. Kindly see the below screenshot for reference.
heading

File: index.html [hunk 13]
```diff
@@ -8,47 +8,324 @@
...
-      <div class="introduction">
-        <h1 class="intro">Hey There.<br />I'm Mucyo</h1>
-        <h2 class="headline-page-h2">I'am a Software Developer</h2>
-
-        <p class="text-1">
-          I can help you build a product, feature or website <br />Look through
-          some of my work and experience!<br />
-          if you like what you see and have a project <br />
-          you need coded, don't hestiate to contacta me.
-        </p>
```
Comment (SajjadAhmad14) [inline index.html:32]: [ OPTIONAL ] Please follow the font color and font family defined in the design.

File: index.html [hunk 14]
```diff
@@ -8,47 +8,324 @@
...
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+          <ul class="links1">
+            <li><a href="#"></a>css</li>
+            <li><a href="#"></a>html</li>
+            <li class="boot"><a href="#"></a>bootstrap</li>
+            <li><a href="#"></a>Ruby</li>
+          </ul>
```
Comment (SajjadAhmad14) [inline index.html:89]: [ OPTIONAL ] Please follow the font color and font family defined in the design.

File: index.html [hunk 15]
```diff
@@ -8,47 +8,324 @@
...
+              hestiate to contacta me.
+            </p>
+          </div>
+
+          <div class="btn_yellow" id="yellow">
+            <button type="submit">Get my Resume</button>
+          </div>
+
+          <div class="line5"></div>
+        </div>
```
Comment (SajjadAhmad14) [inline index.html:195]: [ OPTIONAL ] Please follow the font family defined in the design.

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Mucyo Ntacyonungutse, , 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏
# OPTIONAL
- [ ] Please see inline comments.
## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**